circle.yml

```diff
@@ -0,0 +1,14 @@
+machine:
+  node:
+    version: v0.10.40
+
+
+dependencies:
+  pre:
+    - npm install -g npm@3
+
+test:
+  override:
+    - ./node_modules/.bin/eslint --debug . --format tap | ./node_modules/.bin/tap-xunit > ${CIRCLE_TEST_REPORTS}/lint.xml && test ${PIPESTATUS[0]} -eq 0
+    - NODE_PATH=source ./node_modules/.bin/babel-node source/test | ./node_modules/.bin/tap-xunit > ${CIRCLE_TEST_REPORTS}/test.xml && test ${PIPESTATUS[0]} -eq 0
+    - npm outdated --depth=0
```

Comment:
This is interesting. Have no idea where npm takes those "wanted" versions.

![20151203-014848](https://cloud.githubusercontent.com/assets/175264/11533889/17ff9d10-9960-11e5-80bb-f80672d69f40.png)